### PR TITLE
Add test for azurefile.go

### LINKED ISSUE #1: [warm up] write a few unit tests
As a warm up, could you(@Masquerade0097 ) write a few unit tests first in this project:
`getFileShareInfo`, `getStorageAccount` under https://github.com/andyzhangx/azurefile-csi-driver/blob/master/pkg/azurefile/azurefile.go

You could refer to https://github.com/andyzhangx/azurefile-csi-driver/blob/master/pkg/azurefile/azurefile_test.go#L25-L67

Here is the development guide: 
https://github.com/andyzhangx/azurefile-csi-driver/blob/master/docs/csi-dev.md

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #1 

Added tests for `getFileShareInfo`, `getStorageAccount` under [azurefile.go](https://github.com/andyzhangx/azurefile-csi-driver/blob/master/pkg/azurefile/azurefile.go)

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@andyzhangx The test is passing successfully but the build is failing due to the error in execution of `hack/verify-all.sh`.

**Comment by Masquerade0097** [comment]
@andyzhangx Please review.

File: pkg/azurefile/azurefile_test.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package azurefile
 
 import (
 	"fmt"
+	"github.com/stretchr/testify/assert"
```
Comment (andyzhangx) [inline pkg/azurefile/azurefile_test.go]: Move this line to the bottom:
```
	"fmt"
	"reflect"
	"testing"

	"github.com/stretchr/testify/assert"
```

Comment (Masquerade0097) [reply pkg/azurefile/azurefile_test.go]: Done

**Comment by andyzhangx** [comment]
thanks.
Coverage increased (+3.7%) to 20.694%

**Pull request merged**